enacts/calc.py

```diff
@@ -345,39 +382,72 @@ def cess_date(
 
     Parameters
     ----------
-    soil_moisture : DataArray
-        Array of daily soil moisture.
+    daily_data : DataArray
+        Array of daily soil moisture or daily rainfall.
```

Comment:
My thinking was to test that either et, taw, sminit are all None, either are all non None, and to raise an exception that it should be one or the other and that accordingly, data should be sm or rain. But I am not sure what I could do to check that they do provide the right daily_data when all are None or not None...

Reply:
I think what I had in mind when I made this comment is that maybe users would make fewer mistakes if this function had two separate and mutually exclusive arguments `soil_moisture` and `rainfall`, instead of having a single argument `daily_data` that does double duty.

Minor comment, take it or leave it.

Reply:
How do you do mutually exclusive arguments? Both optional and you check and raise Exception if both (or none) are given? and in that very case, adding a layer of checks with the other optional arguments that belong to the rainfall case.